src/main/java/org/broadinstitute/barclay/help/DefaultDocumentedFeatureHandler.java

```diff
@@ -256,11 +252,15 @@ protected void addCommandLineArgumentBindings(
     }
 
     private void addSummary(final Map<String, Object> root) {
-        final StringBuilder summaryBuilder = new StringBuilder();
-        for (final Tag tag : currentWorkUnit.classDoc.firstSentenceTags()) {
-            summaryBuilder.append(tag.text());
+        // If no summary was found from annotations, use the javadoc if there is any
```

Comment:
Agreed, though I think that the refactoring this code so badly needs should be done in a separate PR since it needs bigger modeling changes (i.e., DocumentedFeatureObject should probably go away completely, and DocWorkUnit should just contain the truth).